src/peerbook/libp2p_peerbook.erl

```diff
@@ -196,7 +195,7 @@ random(Peerbook=#peerbook{tid=TID, store=Store, stale_time=StaleTime}, Exclude0,
                 0 ->
                     rocksdb:iterator_close(Iterator),
                     %% only have one peer
-                    case lists:member(FirstAddr, Exclude) of
+                    case lists:member(rev(FirstAddr), Exclude) of
```

Comment:
what changed here that you needed to rev it?

Reply:
Before we reved the entire exclusion list above on line 189, but that's expensive with 30,000 exclusions.